services/app-api/serverless.yml

```diff
@@ -155,6 +155,15 @@ functions:
           method: get
           cors: true
           authorizer: aws_iam
+  getUserByUsername:
+    handler: handlers/users/get.getUserByUsername
+    role: LambdaApiRole
+    events:
+      - http:
+          path: users/get
+          method: post
```

Comment:
any reason why you have a post? This should be a get method, since we are retrieving info from the db. we agreed to only use get and post methods only.